docs/1.2-DRAFT/introduction.md

```diff
@@ -146,7 +146,7 @@ In RO-Crate Metadata Files, entities are cross-referenced using `@id` reference
 }
 ```
 
-The root is always typed `Dataset` and have several metadata properties that describe the RO-Crate as a whole, as a collection of resources. The section on [root data entity](root-data-entity.md) explores further the required and recommended properties of the root `./`.
+The root is always has `Dataset` as one of its values, though it may have more than one type. It has several metadata properties that describe the RO-Crate as a whole, as a collection of resources. The section on [root data entity](root-data-entity.md) explores further the required and recommended properties of the root `./`.
```

Comment:
"is always has" :arrow_right:  "always has". Also, `Dataset` is one of its types, not values. I propose changing the sentence to:

"The root always has the `Dataset` type, though it may have more than one"